tsconfig.json

```diff
@@ -0,0 +1,41 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "baseUrl": ".",
```

Comment:
Would be better to use ```baseUrl: "./"```? Since as described at the TS Docs, you only need to use ```baseUrl: .``` when setting ```paths``` at your TS config.